File: ALA3/ala3.c
#include<stdio.h>
int main(){
 char str[]="Hello World";
 int i=0, count=0;
 while(str[i]){
 char c=str[i];
 if((c>='a'&&c<='z')||(c>='A'&&c<='Z')){

if(c!='a'&&c!='e'&&c!='i'&&c!='o'&&c!='u'&&

c!='A'&&c!='E'&&c!='I'&&c!='O'&&c!='U')
 count++;
 }
 i++;
 }
 printf("Consonants = %d\n", count);
 return 0;
}

File: ALA3/ala3.py
str1 = "Hello World"

i = 0
count = 0

while i < len(str1):
    c = str1[i]

    # check letter chhe ke nahi
    if (('a' <= c <= 'z') or ('A' <= c <= 'Z')):

        # check vowel nathi (etle consonant)
        if (c != 'a' and c != 'e' and c != 'i' and c != 'o' and c != 'u' and
            c != 'A' and c != 'E' and c != 'I' and c != 'O' and c != 'U'):

            count += 1

    i += 1

print("Consonants =", count)